frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -96,23 +101,27 @@ export const StaticEmbeddingOptionCard = () => {
         )
       }`}
     >
-      <LinkButton
-        variant="default"
-        disabled={!enabled}
-        to={"/admin/settings/embedding-in-other-applications/standalone"}
-      >
-        {t`Manage`}
-      </LinkButton>
+      <Flex align="center" w="100%">
+        <LinkButton
+          variant="default"
+          to={"/admin/settings/embedding-in-other-applications/standalone"}
+        >
+          {t`Manage`}
+        </LinkButton>
+        <Switch
```

Comment:
add a new toggle inside each embedding card.

These are the 2 main changes, nothing else changes in this file.